JSDemos/Demos/Toolbar/Overview/description.md

```diff
@@ -1 +1,35 @@
-The Toolbar is&nbsp;a&nbsp;UI component containing items that usually manage screen content. Those items can be&nbsp;plain text or&nbsp;widgets.
\ No newline at end of file
+The Toolbar contains items that manage the page content: other components and screen view. In this demo, the Toolbar manages the [List](/Documentation/ApiReference/UI_Components/dxList/).
```

Comment:
`other components`

Can we make this more specific and list a few examples ("components such as _____")?

What do you mean by "screen view"? Is this the name of a component as well?

Reply:
I decided to rerwite this sentence